cpp/src/ice2slice/Gen.cpp

```diff
@@ -704,6 +704,7 @@ Gen::TypesVisitor::getOutput(const ContainedPtr& contained)
     {
         string outputName = getOutputName(_fileBase, _modules.size() > 1 ? scope : "");
         unique_ptr<Output> out = make_unique<Output>(outputName.c_str());
+        FileTracker::instance()->addFile(outputName);
```

Comment:
Whenever a Slice compiler creates a file/directory, it should let `FileTracker` know.

For `ice2slice` specifically, it calls `FileTracker::cleanup()` if something goes wrong:
https://github.com/zeroc-ice/ice/blob/ca07ff1899670987053a2d3eb34424c3ff548c2c/cpp/src/ice2slice/Main.cpp#L215-L218 But this call is useless without telling `FileTracker` which files were generated.